src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue

```diff
@@ -1,21 +1,35 @@
 <template>
   <v-item-group mandatory>
-    <v-dialog v-model="dialogVisible" max-width="600">
-      <v-card>
-        <v-card-title>Update Case Status</v-card-title>
+    <v-dialog v-model="dialogVisible" max-width="660">
+      <v-card class="mx-auto">
+        <v-card-title class="ml-2">Update Case Status</v-card-title>
         <v-card-text>
-          Are you sure you want to change the case status from {{ modelValue.status }} to
-          {{ selectedStatus }}
+          Are you sure you want to change the case status from
+          <v-chip :color="statusColors[modelValue.status]">{{ modelValue.status }}</v-chip> to
+          <v-chip :color="statusColors[selectedStatus]">{{ selectedStatus }}</v-chip
+          >?
         </v-card-text>
-        <v-btn
-          class="ml-6 mb-4"
-          size="small"
-          color="info"
-          elevation="1"
-          @click="changeStatus(selectedStatus)"
-        >
-          Submit
-        </v-btn>
+        <v-card-actions class="pt-4">
+          <v-spacer />
+
+          <v-btn @click="changeStatus(selectedStatus)"> Submit </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+
+    <v-dialog v-model="alreadySelectedDialog" max-width="600">
+      <v-card class="mx-auto">
+        <v-card-title class="ml-2">Status Not Changed</v-card-title>
+        <v-card-text>
+          This case was moved to the status
+          <v-chip :color="statusColors[selectedStatus]">{{ selectedStatus }}</v-chip> on
+          {{ selectedStatusTooltip }}
```

Comment:
Do you want to format this better and maybe add in the alt time zone tooltip?

Reply:
By formatting do you mean the code formatting or the format of the text on the page? The former is done for me automatically by the linter.

Reply:
Sorry I left off the important bit: format the time like with `formatToUTC`

Reply:
Rgr, done.